modules/aioseop_sitemap.php

```diff
@@ -3680,6 +4058,18 @@ function modify_post_params_for_external_plugins( &$args ) {
 				$args['suppress_filters'] = false;
 			}
 
+			/**
+			 * Filter to modify the post arguments in case third-party plugins are being used e.g. WPML.
+			 *
+			 * @since ?
+			 *
+			 * @param array $args {
+			 *      Arguments to send to WP_Query.
+			 *
+			 *      @type   string  Argument name.
```

Comment:
Missing variable itself

```
/**
 * Filter to modify the post arguments in case third-party plugins are being used e.g. WPML.
 *
 * @since ?
 *
 * @param array $args {
 *      Arguments to send to WP_Query.
 *
 *      @type string $post_type   Argument name.
 *      @type mixed  $post_status Argument value.
 * }
 */
```